apps/astarte_pairing/test/astarte_pairing/fdo/onboarding/device_service_info_ready_test.exs

```diff
@@ -0,0 +1,99 @@
+#
+# This file is part of Astarte.
+#
+# Copyright 2025 SECO Mind Srl
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+defmodule Astarte.Pairing.FDO.OwnerOnboarding.DeviceServiceInfoReadyTest do
+  use ExUnit.Case, async: true
+
+  alias Astarte.Pairing.FDO.OwnerOnboarding.DeviceServiceInfoReady
+
+  import Astarte.Helpers.FDO
+
+  describe "decode/1" do
+    test "correctly decodes a full valid payload" do
+      hmac = :crypto.strong_rand_bytes(16) |> dbg()
```

Comment:
Let's remove `dbg()` calls from test. Automated tests should not need inspection of values
```suggestion
      hmac = :crypto.strong_rand_bytes(16)
```